packages/design-system-dashboard-cli/package.json

```diff
@@ -26,7 +27,9 @@
   "devDependencies": {
     "eslint": "^7.32.0",
     "husky": ">=6",
+    "jest": "^27.5.1",
     "lint-staged": ">=10",
+    "memfs": "^3.4.1",
```

Comment:
This lets us mock a file system in memory so we can be more controlled in our testing.